pkg/log/log.go

```diff
@@ -61,6 +61,10 @@ func (w *watching) shouldWatch(kind, namespace, name string) bool {
 	return true
 }
 
+func objectDeleted(obj metav1.Object) bool {
```

Comment:
This cans/should be a follow-up (if we do want to do it), but I think this would be a good `z` candidate. What do you think?

Reply:
Ah and here's a link to `z` if you haven't seen it yet. 

https://github.com/acorn-io/z

Reply:
IMO this feels a bit more specialized than what it appears `z` is intended to capture. If in the future this ends up being a more useful utility we could put it in there, but for the time being I think it can stay in this file as an unexported func